docker/celery/Dockerfile

```diff
@@ -23,7 +31,9 @@ COPY ./docker/celery/config.json /cumulus/cumulus/conf/config.json
 
 RUN chown -R celery  /cumulus
 
-RUN git clone https://github.com/Kitware/HPCCloud.git /hpccloud
+RUN git clone https://github.com/Kitware/HPCCloud.git /hpccloud && \
+  cd /hpccloud && \
+  git checkout master
```

Comment:
Is `master` not the default branch?  Just curious if this is actually needed.

Reply:
no, it's not needed, but we want to pin it to a tag eventually